__docs__/wonder-blocks-form/text-field.stories.tsx

```diff
@@ -702,6 +703,68 @@ ReadOnly.parameters = {
     },
 };
 
+export const WithAutofocus: StoryComponentType = () => {
+    const [value, setValue] = React.useState("");
+    const [showDemo, setShowDemo] = React.useState(false);
+
+    const handleChange = (newValue: string) => {
+        setValue(newValue);
+    };
+
+    const handleKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+        if (event.key === "Enter") {
+            event.currentTarget.blur();
+        }
+    };
+
+    const handleShowDemo = () => {
+        setShowDemo(!showDemo);
+    };
+
+    const AutoFocusDemo = () => (
+        <View style={{flexDirection: "row"}}>
+            <Button onClick={() => {}}>Some other focusable element</Button>
+            <TextField
+                id="tf-13"
+                value={value}
+                placeholder="Placeholder"
+                autoFocus={true}
+                onChange={handleChange}
+                onKeyDown={handleKeyDown}
+                style={{flexGrow: 1, marginLeft: 12}}
+            />
+        </View>
+    );
+
+    return (
+        <View>
+            <LabelLarge style={{marginBottom: 12}}>
+                Press the button to view the text field with autofocus.
+            </LabelLarge>
+            <Button
+                onClick={handleShowDemo}
+                style={{width: 300, marginBottom: 24}}
+            >
+                Toggle autoFocus demo
+            </Button>
+            {showDemo && <AutoFocusDemo />}
+        </View>
+    );
+};
+
+WithAutofocus.parameters = {
+    docs: {
```

Comment:
**question:** Do you know if Chromatic generated this snapshot? I see that the GH check says that there are no changes in the tests, but I see that this story is being added.

Wondering to see if there's an issue with Chromatic.

Reply:
I see the snapshots here: https://www.chromatic.com/review?appId=5e1bf4b385e3fb0020b7073c&number=1922&type=linked&view=changes.

Is that what you mean? Or are you referring to something else?

Reply:
Thanks for answering my question offline 🫶